woodwork/statistics_utils/_get_box_plot_info_for_column.py

```diff
@@ -125,6 +175,9 @@ def _get_box_plot_info_for_column(
             "high_indices": high_series.index.tolist(),
         }
 
+    if method == "medcouple":
+        outliers_dict["medcouple"] = mc
```

Comment:
are we ever returning both medcouple and box and whisker? If not, maybe we can just add a type = [medcouple | box] or something like that?

Reply:
It's only one or the other, good idea!